pages/tabular-files/[...id].js

```diff
@@ -134,7 +134,7 @@ TabularFile.propTypes = {
   // Set of documents for file specifications
   fileFormatSpecifications: PropTypes.array.isRequired,
   // ConstructLibraryset this file was integrated in
-  integratedIn: PropTypes.arrayOf(PropTypes.object).isRequired,
+  integratedIn: PropTypes.array.isRequired,
```

Comment:
You can still do `arrayOf(PropTypes.object)` because it’ll either be an array of objects, or an empty array, which is still a (potential) array of objects. React doesn’t have a problem with that. But I’ll leave it up to you whether you want to change that or not.

Reply:
i can change it